x-pack/plugins/index_management/public/application/components/mappings_editor/components/document_fields/fields/fields_list_item.tsx

```diff
@@ -105,6 +109,7 @@ function FieldListItemComponent(
   const indent = treeDepth * CHILD_FIELD_INDENT_SIZE - substractIndentAmount;
 
   const isSemanticText = source.type === 'semantic_text';
+  const inferenceId: string = (source.inference_id as string) ?? ELSER_PRECONFIGURED_ENDPOINTS;
```

Comment:
I am planning to add the preconfigured elser endpoint with the API response so hopefully we will be able to move this code in coming days.

Reply:
yes, however, it renders if `isSemanticText` is true. Not ideal solution but it will be fixed when we add inference_id back to `semantic_text` for default elser endpoint.